### PR TITLE
Lint:Fix

### PR DESCRIPTION
**Pull request by JamieB-gu**
## Why?

For convenience.

## Changes

- Added a `lint:fix` script

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "build:babel": "babel tmp --config-file ./config/babel.config.json --out-dir dist --copy-files",
     "build:cleanup": "rm -rf tmp",
     "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "eslint . --ext .ts,.tsx --fix",
```
Comment (oliverlloyd) [inline package.json:50]:
```suggestion
    "lint:fix": "yarn lint:fix --fix",
```

Does that work?

**Review by oliverlloyd — APPROVED** [review]
Nice idea

**Comment by oliverlloyd** [comment]
PS. I set prettier to fix on save so I've never had a need for this but I can see how that's not everyone's cup of tea

File: package.json (hunk 1 above)
Comment (mxdvl) [reply package.json:50]: Isn't it `yarn lint --fix`?

Comment (JamieB-gu) [reply package.json:50]: I tried that, but I think that passes the `--fix` flag to `yarn` rather than `eslint`.

Comment (mxdvl) [reply package.json:50]: Right, I think you need the `--`, as in `yarn lint -- --fix`.

Comment (JamieB-gu) [reply package.json]: Yup, you're right, good shout. Updated.

**Review by mxdvl — APPROVED** [review]
Even better!

**Pull request merged**